scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -286,7 +287,7 @@ public SchedulerDBManager(Configuration configuration, boolean drop) {
             final boolean finished) {
 
         DBTaskDataParameters parameters = new DBTaskDataParameters(tag, from, to, offset, limit, user,
-                pending, running, finished);
+                pending, running, finished, new SortSpecifierContainer());
```

Comment:
Since such an empty container could be used at several places it would make sense to create it once and store it as a class instance.